CHANGES.md

```diff
@@ -160,6 +160,7 @@ To be released.
  -  `Blockchain<T>` became not to stage transactions having nonce
     less than or equal to the nonce of the same signer's latest
     transaction in already mined blocks.  [[#1173], [#1180]]
+ -  Improved security of `Message.Parse()`  [[#1181]]
```

Comment:
```suggestion
 -  Improved security of `Message.Parse()`.  [[#1181]]
```

It seems also related to https://github.com/planetarium/libplanet/issues/1161, doesn't it?

Reply:
https://github.com/planetarium/libplanet/pull/1181/commits/390689589ce98793466b78fbe444f83fdacc976a amended.